examples/automatic-labelling-folder/variables.tf

```diff
@@ -19,7 +19,7 @@ variable "project_id" {
   description = "The ID of the project to which resources will be applied."
 }
 
-variable "folder_id" {
+variable "sub_folder_id" {
```

Comment:
This change should only be made in the test fixture; `folder_id` is the appropriate name for the example.